### DISCUSSION TITLE
A Logo/Icon and Banner for SPyLib

### DISCUSSION THREAD

**Discussion by lishanl** [SatelCreative/spylib#136] [Ideas]
Dropping ideas here.  

Is it too early to think about adding some spylish logo and banner to the repo for branding / fun?

**Comment by hillairet** [comment]
Definitely not!
I have ideas too ... but I am super interested in your take.  :wink:

**Reply by lishanl** [reply]
I don't have a take. My only idea is to incorporate some pink. 
What are your ideas or anyone else's @SatelCreative/backend ? 😏

**Comment by estellajaysong** [comment]
I made a logo lol 
What do you guys think?
Lines look weird in dark mode but if you click it it'll open in the browser!
@lishanl @hillairet 
![spyliblogo](https://user-images.githubusercontent.com/39111549/169619865-1440b453-3de0-4291-bc27-8c4ad89edd31.png)

**Reply by hillairet** [reply]
So since it's a public repository, I'd prefer not to mess with the Shopify logo ... I'm a tad paranoid when it comes to legal stuff.  :sweat_smile:
